lib/senders/sms.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (log, translator, templates, smsConfig) {
     apiKey: smsConfig.apiKey,
     apiSecret: smsConfig.apiSecret
   })
-  var sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
+  var sendSms = P.promisify(nexmo.message.sendSms, nexmo.message)
```

Comment:
Changed to work with the auth server's old version of bluebird. Can be put back when we fix #1674.